### PR TITLE
feat: parallel workflows, cors proxy

### PR DESCRIPTION
**Pull request by Delusoire**

### CODE REVIEW

File: build/macos/build.sh [hunk 1]
```diff
@@ -3,30 +3,44 @@ set -e
...
+# plutil -remove NSAppleMusicUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSCalendarsUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSCameraUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSContactsUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSHomeKitUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSMicrophoneUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSPhotoLibraryUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSRemindersUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSSiriUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
+# plutil -remove NSSystemAdministrationUsageDescription ./Volume/Spicetify.app/Contents/Info.plist
```
Comment (rxri) [inline build/macos/build.sh:32]:
```suggestion
```

File: cmd/daemon.go [hunk 2]
```diff
@@ -97,7 +104,7 @@ func startDaemon() {
 		}
 		defer watcher.Close()
 
-		log.Println("Watching:", paths.GetSpotifyAppsPath(spotifyDataPath))
+		log.Println("Watcher: watching:", paths.GetSpotifyAppsPath(spotifyDataPath))
```
Comment (rxri) [inline cmd/daemon.go:107]: maybe `[Watcher]`? Or perhaps leave it and I will make log function for every print/errors etc if you don't mind

File: build/macos/build.sh [hunk 3]
```diff
@@ -3,30 +3,44 @@ set -e
...
+
```
Comment (rxri) [inline build/macos/build.sh:33]:
```suggestion

```

File: cmd/daemon.go (hunk 2 above)
Comment (Delusoire) [reply cmd/daemon.go:107]: Let's not make this PR bigger than it needs to be, we can discuss how to do logging separately.

**Pull request merged**